### PR TITLE
more datasets from ISMIR2022

### PR DESCRIPTION
**Pull request by otnemrasordep**

### CODE REVIEW

File: mir-datasets.yaml [hunk 1]
```diff
@@ -783,6 +864,18 @@ LMD - Latin:
   metadata: 10 genres
   contents: 3160 songs
   audio: 'no'
+  
+LocalifyMusicEvents-USA-2019:
+  url: https://github.com/JimiLab/LocalifyMusicEventData
+  metadata: 
+  contentes: 308051 music events that took place in 2019 and from 1139 US cities
```
Comment (stefan-balke) [inline mir-datasets.yaml:871]: please fix the type

Comment (otnemrasordep) [reply mir-datasets.yaml:873]: Apologies, I missed this one. I think it is done now.

Comment (stefan-balke) [reply mir-datasets.yaml:873]: Ah no, it should be `contents` instead of `contentes`

Comment (stefan-balke) [reply mir-datasets.yaml:873]: image
That's also where the unit test is getting an assertion error.

**Comment by stefan-balke** [comment]
Okay, LGTM, thanks for doing it!

**Comment by otnemrasordep** [comment]
My pleasure, sorry for all the mishaps!

**Pull request merged**